src/OpenSage.Game/Data/Ini/IniParser.cs

```diff
@@ -516,11 +516,12 @@ public LazyAssetReference<ObjectCreationList> ParseObjectCreationListReference()
         public LazyAssetReference<Graphics.Animation.W3DAnimation>[] ParseAnimationReferenceArray()
         {
             var result = new List<LazyAssetReference<Graphics.Animation.W3DAnimation>>();
-
             IniToken? token;
             while ((token = GetNextTokenOptional()).HasValue)
             {
-                result.Add(_assetStore.ModelAnimations.GetLazyAssetReferenceByName(token.Value.Text));
+                var name = token.Value.Text;
+                if (!name.Contains('.')) name = ((ModelConditionState)Temp).Skeleton + "." + name;
+                result.Add(_assetStore.ModelAnimations.GetLazyAssetReferenceByName(name));
```

Comment:
sometimes models reference their animation like 'AnimationName' which did not work before and sometimes they are referenced like 'SkeletonName.AnimationName'